package.json

```diff
@@ -150,10 +150,10 @@
     "json-loader": "0.5.7",
     "mini-css-extract-plugin": "2.7.5",
     "msw": "0.47.4",
-    "nock": "13.2.4",
+    "nock": "13.5.4",
```

Comment:
It looks like we're not directly using `nock`. My understanding is that nested deps will show up in `yarn.lock` and not the `package.json`. However, when I `yarn remove nock`, I no longer see it as a nested dep in the `yarn.lock`, which (again) I think it would be if it were a nested dep. Are you sure this package is being used anywhere? I think we can just remove it.

Reply:
@fleetdm/frontend feel free to set me straight here

Reply:
Yes, you're right. I don't see it being used, and I removed it.

FYI: I filed a related issue to regularly update our dependencies: https://github.com/fleetdm/fleet/issues/18057